src/Poseidon/SequencingSource.hs

```diff
@@ -293,23 +420,30 @@ readSeqSourceFileRow seqSourcePath (lineNumber, row) = do
                 Right (pS :: SeqSourceRow) -> do return $ Right pS
 
 -- SeqSource consistency checks
-
 checkSeqSourceConsistency :: FilePath -> SeqSourceRows -> Either PoseidonException SeqSourceRows
-checkSeqSourceConsistency seqSourcePath xs
-    | not $ checkSamplesUnique xs = Left $ PoseidonFileConsistencyException seqSourcePath
-        "The values in the sample_accession column are not unique"
+checkSeqSourceConsistency _ xs
+    -- | ... no tests implemented
```

Comment:
OK, I suppose this is now empty because of the lack of unique keys, and lack of explicit foreign-key constraints? Do you want to keep those functions for later purpose?

Reply:
Yes - I thought it would be fine to keep them around for the future.